### PR TITLE
Fix for supporting enums without eslint complaining about them

### PR DESCRIPTION
**Pull request by ricotijsen**
https://github.com/typescript-eslint/typescript-eslint/issues/2483

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -49,5 +49,6 @@ module.exports = {
 			groups: ['builtin', 'external', 'parent', 'sibling', 'index'],
 			alphabetize: { order: 'asc' },
 		}],
+		'no-shadow': ['off'],
```
Comment (jdpnielsen) [inline index.js:52]: I think this rule should only be turned off in the typescript lint rules - if for some reason this lint config is used for a javascript project the no-shadow rule will be completely disabled.

**Review by jdpnielsen — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (ricotijsen) [reply index.js:52]: Somehow it really wants it: 
image

Comment (ricotijsen) [reply index.js:52]: Or did you mean moving it to the typescript.js file only?

File: index.js [hunk 2]
```diff
@@ -48,6 +48,6 @@ module.exports = {
 			'newlines-between': 'always',
 			groups: ['builtin', 'external', 'parent', 'sibling', 'index'],
 			alphabetize: { order: 'asc' },
-		}],
+		}],
```
Comment (jdpnielsen) [inline index.js:51]:
```suggestion
		}],
```

**Review by jdpnielsen — CHANGES_REQUESTED** [review]

**Review by jdpnielsen — APPROVED** [review]

**Pull request merged**